src/ipc/ipc-helper.c

```diff
@@ -180,19 +179,25 @@ int comp_verify_params(struct comp_dev *dev, uint32_t flag,
 int comp_buffer_connect(struct comp_dev *comp, uint32_t comp_core,
 			struct comp_buffer *buffer, uint32_t dir)
 {
```

Comment:
Can we merge comp_buffer_connect and ipc_comp_to_buffer_connect & ipc_buffer_to_comp_connect in ipc3/ipc-helper ?  Last time I want but I get some unknow bugs for multi-core test. Maybe it our core setting is not inconstant.

Reply:
maybe in another PR @RanderWang if thats ok